packages/opentelemetry-core/src/context/propagation/HttpTraceContext.ts

```diff
@@ -15,12 +15,13 @@
  */
 
 import {
-  Carrier,
   Context,
   HttpTextFormat,
   SpanContext,
   TraceFlags,
 } from '@opentelemetry/api';
+import { GetterFunction } from '@opentelemetry/api';
+import { SetterFunction } from '@opentelemetry/api';
```

Comment:
I would combine these (`GetterFunction` and `SetterFunction`) imports into top one, otherwise this can make it hard to determine the aggregate imports of a file/module.